### PR TITLE
[Console] Add option "--implements" to "make:class" command

### PR DESCRIPTION
**Pull request by nguyentien98**
## About

- N/A

## References

- N/A

### CODE REVIEW

File: src/Console/DecorateStub.php [hunk 1]
```diff
@@ -20,30 +20,62 @@ trait DecorateStub
...
+        $name = $this->getClassName($name);
+        $parentName = $this->getClassName($parent);
         $stub = str_replace(
             [
                 'DummyNamespace',
                 'DummyClass',
                 'DummyParentFullName',
                 'DummyParentName',
+                'DummyInterfaces',
+                'ImportInterfaces',
```
Comment (xuanquynh) [inline src/Console/DecorateStub.php]: Rename `ImportInterfaces` to `DummyImportInterfaces`!

File: src/Console/DecorateStub.php [hunk 2]
```diff
@@ -20,30 +20,62 @@ trait DecorateStub
...
+            } else {
+                $importInterfaces = sprintf('use %s;', $interfaces);
+                $interfaceNames .= $this->getClassName($interfaces);
+            }
+        }
 
+        $namespace = $this->getNamespace($name);
+        $name = $this->getClassName($name);
+        $parentName = $this->getClassName($parent);
         $stub = str_replace(
```
Comment (xuanquynh) [inline src/Console/DecorateStub.php]: Use `strtr` function!

```php
// Example
echo strtr('a b', ['a' => 'a1', 'b' => 'b1']);
```

**Comment by xuanquynh** [comment]
https://github.com/sepiphy/laravel-extensions/pull/21

**Comment by xuanquynh** [comment]
@nguyentien98 Why don't you use this syntax?

```php
strtr('some string', [
    'first' => 'Thu Nhat',
    'second' => 'Thu Hai',
]);
```

**Comment by nguyentien98** [comment]
@xuanquynh Thanks. I've edited my code. Please re-review. :dancing_men:

**Comment by xuanquynh** [comment]
@nguyentien98 Thanks!

**Pull request merged**